docker/packager/packager

```diff
@@ -104,32 +105,45 @@ def run_vagrant_box_with_env(image_path, output_dir, ch_root):
 
 def parse_env_variables(build_type, compiler, sanitizer, package_type, cache, distcc_hosts, unbundled, split_binary, version, author, official):
     result = []
+    cmake_flags = ' $CMAKE_FLAGS '
```

Comment:
Maybe easer to make it `list`, add elements with `cmake_flags.append` and after join to string with `' '.join(cmake_flags)`?